src/main/java/mate/academy/model/Ticket.java

```diff
@@ -0,0 +1,53 @@
+package mate.academy.model;
+
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import javax.persistence.ManyToOne;
+import javax.persistence.Table;
+
+@Entity
+@Table(name = "tickets")
+public class Ticket {
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Long id;
+    @ManyToOne
+    private MovieSession movieSession;
+    @ManyToOne
```

Comment:
You use join fetch in ShopCartDao , that is why you don't need EAGER here.
```suggestion
    @ManyToOne(fetch = FetchType.LAZY)
    private MovieSession movieSession;
    @ManyToOne(fetch = FetchType.LAZY)
```